examples/v1beta1/pipelineruns/no-ci/pipeline-timeout.yaml

```diff
@@ -53,3 +53,62 @@ spec:
       value: "Good Morning, Bob!"
     - name: NIGHT_GREETINGS
       value: "Good Night, Bob!"
+
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: pipelinerun-timeouts
+spec:
+  # 1 hour and half timeout for the pipeline
+  # 1 hour and fifteen minutes for the pipeline tasks
+  # 15 minutes for the finally tasks
+  timeouts:
```

Comment:
Suggesting adding a comment that this field requires `enable-api-fields: alpha` flag.  Just in case a user comes along and copy/pastes this into their own pipelineruns.